### PR TITLE
PICARD-1206: force style for GroupBox::title

### PR DESCRIPTION
**Pull request by zas**
It is a workaround for a Qt bug, resulting on partially truncated group box titles.

# Summary

* This is a…
    * [x] Bug fix
    * [ ] Feature addition
    * [ ] Refactoring
    * [ ] Minor / simple change (like a typo)
    * [ ] Other
* **Describe this change in 1-2 sentences**:

# Problem

* JIRA ticket (_optional_): [PICARD-1206](https://tickets.metabrainz.org/browse/PICARD-1206)

# Solution

# Action

### CODE REVIEW

File: picard/tagger.py [hunk 1]
```diff
@@ -118,7 +118,14 @@ def __init__(self, picard_args, unparsed_args, localedir, autoupdate):
         # Allow High DPI Support
         self.setAttribute(QtCore.Qt.AA_UseHighDpiPixmaps)
         self.setAttribute(QtCore.Qt.AA_EnableHighDpiScaling)
-
+        super().setStyleSheet(
```
Comment (sambhav) [inline picard/tagger.py]: Can't really test this as it doesn't affect me. Maybe @vishichoudhary can test and see if it fixes it for him too?

Comment (vishichoudhary) [reply picard/tagger.py]: @zas @sambhav no effect of adding these two
```
 subcontrol-origin: margin;
 subcontrol-position: top left;
```
this too works for me 

```
 super().setStyleSheet(
             """ 
             QGroupBox::title { }
             """
) 
```

weird.

Comment (zas) [reply picard/tagger.py]: @vishichoudhary : i changed the patch to use just this, it works for me too.

Comment (sambhav) [reply picard/tagger.py]: Isn't this a bit too weird and unpredictable?

Comment (zas) [reply picard/tagger.py]: Yes it is... but it works ... don't ask me why.

**Comment by sambhav** [comment]
@zas the new setStyle causes other bugs. See the following (Picard.ini is same for both but the PR causes the saved sizes to not reflect).

https://i.imgur.com/GwPIGHE.png
https://i.imgur.com/aOpqwX9.png

**Comment by zas** [comment]
> @zas the new setStyle causes other bugs. See the following (Picard.ini is same for both but the PR causes the saved sizes to not reflect).

What is the issue ? Size of the dialog box ?

**Comment by zas** [comment]
> I found the fix a bit hacky.

That's clearly an hacky workaround, fix has to be done upstream (Qt).

**Pull request merged**